packages/moonstone/Scrollable/ScrollButtons.js

```diff
@@ -66,6 +66,16 @@ class ScrollButtons extends Component {
 		 */
 		disabled: PropTypes.bool,
 
+		/**
+		 * When `true`, allows 5-way navigation to the scrollbar controls. By default, 5-way will
+		 * not move focus to the scrollbar controls.
+		 *
+		 * @type {Boolean}
+		 * @default false
+		 * @public
+		 */
+		focusableScrollbar: PropTypes.bool,
```

Comment:
How about just `focusable`?

Reply:
I'll change it to `focusableScrollButtons`.